src/components/Home/Header/index.tsx

```diff
@@ -53,54 +63,54 @@ const cliCaptionData: Array<{ bold: string; text: string }> = [
   }
 ]
 
-const logosData: Array<{
-  src: string
-  widthSm: number
-  widthMd: number
-  widthLg: number
-  alt: string
-}> = [
+const logosData: Array<ILogo> = [
   {
-    widthSm: 66,
-    widthMd: 88,
-    widthLg: 136,
+    widthSm: 74,
+    widthMd: 96,
+    widthLg: 144,
     src: fastapiLogo,
-    alt: 'FastAPI logo'
+    alt: 'FastAPI logo',
+    link: '/doc/user-guide/serving/fastapi'
```

Comment:
Thanks a lot @yathomasi! I see the border is around logos, can we make them around boxes for logos maybe? Otherwise it looks inconsistent somehow
<img width="1020" alt="image" src="https://user-images.githubusercontent.com/6797716/209930467-13e7821d-8a1b-4b67-a7b6-c7ac184a4678.png">
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/6797716/209930496-2987edc3-72f3-4604-a913-8ff9935b3308.png">

Reply:
Yes, great suggestion. I have updated it a bit. Let me how you think.

Reply:
Thanks! Looks much better! Could we also make them always the same height? Now this is how it looks for me:
<img width="238" alt="image" src="https://user-images.githubusercontent.com/6797716/210037357-ebda6fd8-870f-4a3f-98f4-a18f51e62b11.png">
and 
<img width="252" alt="image" src="https://user-images.githubusercontent.com/6797716/210037370-e0d547bb-dc54-496f-b8fb-d7985f7b5170.png">

Reply:
Thanks a ton! Works as expected now! Now I need to get feedback from @iterative/mlem and update "Working with models" page.